### PR TITLE
[SEM - 027]: Tooltips

### PR DESCRIPTION
**Pull request by GKakuuu**
Se modificó el package.json para poder ocupar librerías tales como:

- react-icons : iconos de react, necesarios para ser más precisos estéticamente en lo solicitado en el ticket
- @testing-library/user-event : para poder realizar las pruebas en Vitest controlando eventos del usuario en el componente

### CODE REVIEW

File: src/ui/components/atoms/Tooltip/Tooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { Tooltip } from './Tooltip';
+
+// Configuración del Storybook
+const meta: Meta<typeof Tooltip> = {
+  title: 'ui/components/atoms/Tooltip',
+  component: Tooltip,
+  argTypes: {
+    text: {
+      control: 'text',
+      description: 'Texto que se mostrará al usuario para interactuar.',
```
Comment (Ditmar) [inline src/ui/components/atoms/Tooltip/Tooltip.stories.tsx:11]: Please change the language of your description test to English

File: src/ui/components/atoms/Tooltip/Tooltip.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    },
+    toolTipText: {
+      control: 'text',
+      description: 'Texto que aparecerá en el tooltip.',
```
Comment (Ditmar) [inline src/ui/components/atoms/Tooltip/Tooltip.stories.tsx:15]: Please change the language of your description test to English

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice work! please i left some comments check it review it and your pr will be approved

File: src/ui/components/atoms/Tooltip/Tooltip.stories.tsx (hunk 1 above)
Comment (GKakuuu) [reply src/ui/components/atoms/Tooltip/Tooltip.stories.tsx]: Description changes successfully completed

File: src/ui/components/atoms/Tooltip/Tooltip.stories.tsx (hunk 2 above)
Comment (GKakuuu) [reply src/ui/components/atoms/Tooltip/Tooltip.stories.tsx]: Description changes successfully completed

**Comment by GKakuuu** [comment]
**Link Storybooks:**
[](https://ditmar.github.io/backoffice-openscience/branch-SEM-027)

**Path of the component:**
[](https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-027/src/ui/components/atoms/Tooltip)

File: src/ui/components/atoms/Tooltip/Tooltip.module.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```
Comment (Ditmar) [inline src/ui/components/atoms/Tooltip/Tooltip.module.scss:1]: please change the approach, use embed fonts instead .

File: src/ui/components/atoms/Tooltip/Tooltip.module.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  font-size: 12px;
+  margin-left: 8px;
+}
+
+/* Estilo del texto principal */
+.text {
+  font-size: 12px;
+}
+
+/* Estilo del texto que aparece en el tooltip (inicialmente oculto) */
```
Comment (Ditmar) [inline src/ui/components/atoms/Tooltip/Tooltip.module.scss:30]: remove all comments

File: src/ui/components/atoms/Tooltip/Tooltip.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        options: ['small', 'medium', 'large'],
+      },
+      description: 'Determines the size of the tooltip.',
+    },
+  },
+};
+
+export default meta;
+
+// Historias
```
Comment (Ditmar) [inline src/ui/components/atoms/Tooltip/Tooltip.stories.tsx:29]: remove the comments

**Review by Ditmar — CHANGES_REQUESTED** [review]
please read the comments

**Comment by GKakuuu** [comment]
[**Link Storybooks**](https://ditmar.github.io/backoffice-openscience/branch-SEM-027/?path=/story/ui-components-atoms-tooltip--small)


[**Path of the component**](https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-027/src/ui/components/atoms/Tooltip)

**Comment by GKakuuu** [comment]
The requested changes were made.

[**Link Storybooks**](https://ditmar.github.io/backoffice-openscience/branch-SEM-027/?path=/story/ui-components-atoms-tooltip--small)


[**Path of the component**](https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-027/src/ui/components/atoms/Tooltip)

**Review by Ditmar — APPROVED** [review]

**Pull request merged**